### DISCUSSION TITLE
calendar and newsletters

### DISCUSSION THREAD

**Discussion by j-p-courneya** [Relay-Maryland/relaymaryland.com#7] [General]
Calendar: [calendar](https://calendar.google.com/calendar/u/2?cid=cmVsYXlpbXByb3ZlbWVudGFzc29jaWF0aW9uQGdtYWlsLmNvbQ)

newsletters: [newsletters](https://drive.google.com/drive/folders/1W9jw8-M25uZ9zGZnVsoD7iyEmvYFywae?usp=sharing)

**Comment by brianzelip** [comment]
@j-p-courneya can you add the March covered dish event info here for me to play with?

**Reply by j-p-courneya** [reply]
{Covered Dish} Sun 17 MAR | 4-7pm | 1710 Arlington Ave | Relay Town Hall

**Comment by j-p-courneya** [comment]
@brianzelip 

## Ex. 1

### Subject

Petition Signing-March 17th 4-7pm

### Body

Dear Relay Neighbors:

We invite you to sign a Petition opposing CZMP 1-008 & 1-009 during our "Covered Dish" on March 17th, 4-7pm at the Relay Town Hall; 1710 Arlington Avenue.

Issue CZMP 1-008 & 1-009 state that the property owner wishes to re-zone properties from RC-5 (Rural Conservation) to BM (Business) zoning.  While these properties are separated by I-95; they are linked and along with Gun Road residents, Relay residents voiced their opposition to this zoning request change on February 6th during the County Planning process.  This area was deemed rural conservation for some time now and for good reason.  We have to band together and ensure the zoning remains intact.

Neighbors from Gun Road have submitted their petition to Councilman Pat Young's office and plan to do the same.  We ask if you are like-minded that you sign this petition.  If you are unable to join us this coming Sunday, please submit your opposition to relaytownhall@gmail.com.

Thank you for your continued support and hope to see you on Sunday.


## Ex. 2

### Subject

{Covered Dish} Sun 17 MAR | 4-7pm | 1710 Arlington Ave | Relay Town Hall

### Body

![Screenshot_20240313-114929.png](https://github.com/Relay-Maryland/relaymaryland.com/assets/11720131/10f7ce26-f72d-41ca-8979-cb77cbc0cccf)


If you haven't already renewed your RIA membership you can do so at the covered dish.

We will also have a petition circulating for CZMP 1-008 & 1-009 opposing the zoning change.

See you at 4:00PM St. Patricks Day.

**Comment by j-p-courneya** [comment]
## Ex. 1

### Subject

Petition Signing-March 17th 4-7pm

### Body

Dear Relay Neighbors:

We invite you to sign a Petition opposing CZMP 1-008 & 1-009 during our "Covered Dish" on March 17th, 4-7pm at the Relay Town Hall; 1710 Arlington Avenue.

Issue CZMP 1-008 & 1-009 state that the property owner wishes to re-zone properties from RC-5 (Rural Conservation) to BM (Business) zoning.  While these properties are separated by I-95; they are linked and along with Gun Road residents, Relay residents voiced their opposition to this zoning request change on February 6th during the County Planning process.  This area was deemed rural conservation for some time now and for good reason.  We have to band together and ensure the zoning remains intact.

Neighbors from Gun Road have submitted their petition to Councilman Pat Young's office and plan to do the same.  We ask if you are like-minded that you sign this petition.  If you are unable to join us this coming Sunday, please submit your opposition to relaytownhall@gmail.com.

Thank you for your continued support and hope to see you on Sunday.


## Ex. 2

### Subject

{Covered Dish} Sun 17 MAR | 4-7pm | 1710 Arlington Ave | Relay Town Hall

### Body

![Screenshot_20240313-114929.png](https://github.com/Relay-Maryland/relaymaryland.com/assets/11720131/10f7ce26-f72d-41ca-8979-cb77cbc0cccf)


If you haven't already renewed your RIA membership you can do so at the covered dish.

We will also have a petition circulating for CZMP 1-008 & 1-009 opposing the zoning change.

See you at 4:00PM St. Patricks Day.

**Comment by j-p-courneya** [comment]
@brianzelip 

I noticed we don't have a spot for the quarterly newsletter. I would have a PDF available should I use the same approach as the membership form and just set up a link? What if I want like an archive of newsletter? Does that require much configuration? The newsletter would live in RIA?

**Comment by j-p-courneya** [comment]
@brianzelip 

Let's add the newsletter to the RIA page at the bottom. Were you thinking that it would be like Zions embedded pdf/newsletter?

**Reply by brianzelip** [reply]
Sounds good. Yes that is what I was thinking.
